google/cloud/pubsub/internal/tracing_message_batch.cc

```diff
@@ -35,66 +35,86 @@ void TracingMessageBatch::SaveMessage(pubsub::Message m) {
       opentelemetry::context::RuntimeContext::GetCurrent());
   active_span->AddEvent("gl-cpp.added_to_batch");
   {
-    std::lock_guard<std::mutex> lk(mu_);
+    std::lock_guard<std::mutex> lk(message_mu_);
     message_spans_.push_back(std::move(active_span));
   }
   child_->SaveMessage(std::move(m));
 }
 
-void TracingMessageBatch::AddMessageSpanMetadata() {
-  for (auto& message_span : message_spans_) {
+void AddMessageSpanMetadata(
```

Comment:
nit: this should go to the anoymous namespace in this `.cc` file then.

Reply:
DOne